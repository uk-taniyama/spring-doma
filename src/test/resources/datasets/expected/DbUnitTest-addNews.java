DataSetBuilder builder = new DataSetBuilder();
IDataSet dataSet = builder
    .table("MST_NEWS")
    .row()
        .column("MST_NEWS_ID", 1)
        .column("ROLE_ID", "ROLE_ADMIN")
        .column("SUBJECT", "表題テスト1")
        .column("URL", "http://hoge/test1")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 2)
        .column("ROLE_ID", "ROLE_ADMIN")
        .column("SUBJECT", "表題テスト2")
        .column("URL", "http://hoge/test2")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 3)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト1")
        .column("URL", "http://hogehoge/test1")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 4)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト2")
        .column("URL", "http://hogehoge/test2")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 5)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト3")
        .column("URL", "http://hogehoge/test3")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 6)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト4")
        .column("URL", "http://hogehoge/test4")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 7)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト5")
        .column("URL", "http://hogehoge/test5")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 8)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト6")
        .column("URL", "http://hogehoge/test6")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 9)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト7")
        .column("URL", "http://hogehoge/test7")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 10)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト8")
        .column("URL", "http://hogehoge/test8")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 11)
        .column("ROLE_ID", "ROLE_USER")
        .column("SUBJECT", "テスト表題テスト9")
        .column("URL", "http://hogehoge/test9")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "null")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null")
    .row()
        .column("MST_NEWS_ID", 12)
        .column("ROLE_ID", "100")
        .column("SUBJECT", "表題100")
        .column("URL", "http://a.b")
        .column("VERSION", 0)
        .column("INSERT_USER", "null")
        .column("INSERT_DATE", "2022-11-28 01:39:14.733535")
        .column("UPDATE_USER", "null")
        .column("UPDATE_DATE", "null").build();