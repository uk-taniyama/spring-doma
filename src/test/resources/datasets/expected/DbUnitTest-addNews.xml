<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <MST_NEWS MST_NEWS_ID="1" ROLE_ID="ROLE_ADMIN" SUBJECT="表題テスト1" URL="http://hoge/test1" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="2" ROLE_ID="ROLE_ADMIN" SUBJECT="表題テスト2" URL="http://hoge/test2" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="3" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト1" URL="http://hogehoge/test1" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="4" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト2" URL="http://hogehoge/test2" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="5" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト3" URL="http://hogehoge/test3" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="6" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト4" URL="http://hogehoge/test4" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="7" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト5" URL="http://hogehoge/test5" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="8" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト6" URL="http://hogehoge/test6" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="9" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト7" URL="http://hogehoge/test7" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="10" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト8" URL="http://hogehoge/test8" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="11" ROLE_ID="ROLE_USER" SUBJECT="テスト表題テスト9" URL="http://hogehoge/test9" VERSION="0"/>
  <MST_NEWS MST_NEWS_ID="12" ROLE_ID="100" SUBJECT="表題100" URL="http://a.b" VERSION="0" INSERT_DATE="2022-11-28 01:39:14.733535"/>
</dataset>
